specificatie/verblijfplaats.yaml

```diff
@@ -0,0 +1,392 @@
+    Postcode:
+      description: |
+        De door PostNL vastgestelde code die bij een bepaalde combinatie van een straatnaam en een huisnummer hoort.
+      type: string
+      pattern: ^[1-9]{1}[0-9]{3}[ ]?[A-Za-z]{2}$
+      example: "2341SX"
+    Straat:
+      type: string
+      pattern: ^[a-zA-Z0-9À-ÿ '\-\']{1,80}$
+      example: "Tulpstraat"
+    VertrokkenOnbekendWaarheen:
+      type: boolean
+      description: |
+        Indicatie dat de persoon is vertrokken naar het buitenland, maar dat niet bekend is waar naar toe.
+    Woonplaats:
+      description: |
+        Een woonplaats is een gedeelte van het grondgebied van de gemeente met een naam.
+      title: woonplaats naam
+      type: string
+      pattern: ^[a-zA-Z0-9À-ÿ '\-\']{1,80}$
+      example: "Duiven"
+    AbstractVerblijfplaatsBeperkt:
+      type: object
+      description: |
+        Gegevens over het verblijf of de woonlocatie van een persoon.
+      required:
+        - type
+      properties:
+        type:
+          type: string
+      discriminator:
+        propertyName: type
+        mapping:
+          GbaVerblijfplaats: '#/components/schemas/GbaVerblijfplaatsBeperkt'
+          VerblijfplaatsBuitenland: '#/components/schemas/VerblijfplaatsBuitenlandBeperkt'
+          Verblijfplaats: '#/components/schemas/VerblijfplaatsBeperkt'
+          VerblijfOnbekend: '#/components/schemas/VerblijfplaatsOnbekend'
+          Locatie: '#/components/schemas/LocatieBeperkt'
+    VerblijfplaatsBeperkt:
+      allOf:
+        - $ref: '#/components/schemas/AbstractVerblijfplaatsBeperkt'
+        - type: object
+          properties:
+            functieAdres:
+              $ref: 'https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/v1.2.0/api-specificatie/common.yaml#/components/schemas/Waardetabel'
+            adresregel1:
+              $ref: '#/components/schemas/Adresregel1'
+            adresregel2:
+              $ref: '#/components/schemas/Adresregel2'
+            woonplaats:
+              $ref: '#/components/schemas/Woonplaats'
+    VerblijfplaatsBuitenlandBeperkt:
+      allOf:
+        - $ref: '#/components/schemas/AbstractVerblijfplaatsBeperkt'
+        - type: object
+          properties:
+            adresregel1:
+              $ref: '#/components/schemas/Adresregel1'
+            adresregel2:
+              $ref: '#/components/schemas/Adresregel2'
+            adresregel3:
+              $ref: '#/components/schemas/Adresregel3'
+            land:
+              $ref: 'land.yaml#/components/schemas/AbstractLand'
+    VerblijfplaatsOnbekendBeperkt:
+      allOf:
+        - $ref: '#/components/schemas/AbstractVerblijfplaatsBeperkt'
+        - type: object
+          properties:
+            vertrokkenOnbekendWaarheen:
+              $ref: '#/components/schemas/VertrokkenOnbekendWaarheen'
+    LocatieBeperkt:
+      allOf:
+        - $ref: '#/components/schemas/AbstractVerblijfplaatsBeperkt'
+        - type: object
+          properties:
```

Comment:
hierin ontbreken nu adresregel1 en adresregel2. Die moeten hier ook beschikbaar zijn, zodat altijd, ongeacht het type (behalve GbaVerblijfplaats) adresregel1 en adresregel2 gebruikt kan worden. Zeker bij beperkt is het super makkelijk voor een maken van een GUI dat in de lijst met alle gevonden personen voor alle personen het adres gevonden kan worden uit dezelfde twee velden.

Reply:
done